proposed/event-dispatcher.md

```diff
@@ -88,11 +90,11 @@ A Listener Provider is a service object responsible for determining what Listene
 * Extracting some information from an object referenced by the Event, such as an Entity, and calling pre-defined lifecycle methods on that object.
 * Delegating its responsibility to one or more other Listener Providers using some arbitrary logic.
 
-Or some combination of those, or some other mechanism as desired.
+Any combination of the above, or other mechanisms, MAY be used as desired.
```

Comment:
I think the proposed changes make the intention more clear here: providers have great leeway in how they identify relevant listeners.